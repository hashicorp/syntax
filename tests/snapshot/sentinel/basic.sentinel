hour = 4

past_midnight = rule { hour >= 0 }
before_noon   = rule { hour < 12 }

main = rule {
    past_midnight and
    before_noon
}
