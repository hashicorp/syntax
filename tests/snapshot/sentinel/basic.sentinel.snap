>hour = 4
#^^^^^ source.sentinel
#     ^ source.sentinel keyword.operator.symbol.sentinel
#      ^ source.sentinel
#       ^ source.sentinel constant.numeric.number.sentinel
>
>past_midnight = rule { hour >= 0 }
#^^^^^^^^^^^^^^ source.sentinel
#              ^ source.sentinel keyword.operator.symbol.sentinel
#               ^ source.sentinel
#                ^^^^ source.sentinel keyword.other.sentinel
#                    ^^^^^^^^ source.sentinel
#                            ^ source.sentinel keyword.operator.symbol.sentinel
#                             ^ source.sentinel keyword.operator.symbol.sentinel
#                              ^ source.sentinel
#                               ^ source.sentinel constant.numeric.number.sentinel
#                                ^^^ source.sentinel
>before_noon   = rule { hour < 12 }
#^^^^^^^^^^^^^^ source.sentinel
#              ^ source.sentinel keyword.operator.symbol.sentinel
#               ^ source.sentinel
#                ^^^^ source.sentinel keyword.other.sentinel
#                    ^^^^^^^^ source.sentinel
#                            ^ source.sentinel keyword.operator.symbol.sentinel
#                             ^ source.sentinel
#                              ^^ source.sentinel constant.numeric.number.sentinel
#                                ^^^ source.sentinel
>
>main = rule {
#^^^^^ source.sentinel
#     ^ source.sentinel keyword.operator.symbol.sentinel
#      ^ source.sentinel
#       ^^^^ source.sentinel keyword.other.sentinel
#           ^^^ source.sentinel
>    past_midnight and
#^^^^^^^^^^^^^^^^^^ source.sentinel
#                  ^^^ source.sentinel keyword.control.sentinel
>    before_noon
#^^^^^^^^^^^^^^^^ source.sentinel
>}
#^^ source.sentinel
>